dynamodb/src/test/scala/akka/stream/alpakka/dynamodb/ItemSpec.scala

```diff
@@ -20,59 +26,67 @@ class ItemSpec extends TestKit(ActorSystem("ItemSpec")) with AsyncWordSpecLike w
   implicit val materializer: ActorMaterializer = ActorMaterializer()
   implicit val ec: ExecutionContext = system.dispatcher
 
-  override def beforeAll(): Unit = {
-    System.setProperty("aws.accessKeyId", "someKeyId")
-    System.setProperty("aws.secretKey", "someSecretKey")
+  implicit val client: DynamoDbAsyncClient = DynamoDbAsyncClient
+    .builder()
+    .region(Region.AWS_GLOBAL)
+    .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials.create("x", "x")))
+    .endpointOverride(new URI("http://localhost:8001/"))
+    .build()
+
+  override def afterAll(): Unit = {
+    client.close()
+    shutdown()
+    super.afterAll()
   }
 
   "DynamoDB" should {
 
     import ItemSpecOps._
 
     "1) list zero tables" in assertAllStagesStopped {
-      DynamoDb.single(listTablesRequest).map(_.getTableNames.asScala.count(_ == tableName) shouldBe 0)
+      DynamoDb.single(listTablesRequest).map(_.tableNames().asScala shouldBe empty)
```

Comment:
Here and below `()` can be omitted in Scala. In fact usually they are omitted for methods without parameters, if the method does not side-effect.

Reply:
Sure. My IDE auto completes with parenthesis on java methods. Will clean this